src/invidious/channels.cr

```diff
@@ -355,14 +355,19 @@ def fetch_channel(ucid, db, pull_all_videos = true, locale = nil)
   return channel
 end
 
-def fetch_channel_playlists(ucid, author, auto_generated, continuation, sort_by)
-  if continuation || auto_generated
-    url = produce_channel_playlists_url(ucid, continuation, sort_by, auto_generated)
+def fetch_channel_playlists(ucid, author, continuation, sort_by)
+  if continuation
+    response_json = request_youtube_api_browse(continuation)
+    result = JSON.parse(response_json.match(/"continuationItems": (?<items>\[.*\]),/m).try &.["items"] || "{}")
```

Comment:
Please parse this to JSON and access it that way. `request_youtube_api_browse` should probably return JSON directly. Is it possible to get anything but JSON from `/youtubei/v1/browse`?

Reply:
it does return JSON directly
it's a (bad) habit of mine to use regex for everything, my bad

Reply:
> Is it possible to get anything but JSON from `/youtubei/v1/browse`?

Nope. The request headers contain `content-type: application/json`, so even in the case of an error, JSON is returned. Reminds me to open an issue, because there are still some code that expects HTML there (see changes I made in #1911)

Reply:
@SamantazFox That's great, please change `request_youtube_api_browse` to return parsed JSON and check for errors if possible.

Reply:
@saltycrys Sure, I'll work on it :)